package.json

```diff
@@ -129,6 +129,9 @@
     "printWidth": 120
   },
   "license": "Apache-2.0",
+  "grafana": {
+    "chromium_revision": "533271"
```

Comment:
what's the reason for this?

Reply:
We need to have a version we want to bundle, I copied the version which puppeteer is currently using. The reason why it is in `package.json` is because I need to put it somewhere where I can access it from grunt.